spider.js

```diff
@@ -0,0 +1,138 @@
+var request = require('superagent');
+var url = require('url');
+var jsdom = require('jsdom');
+var app = require('./app');
+var server;
+var serveraddress = "http://localhost:" + app.PORT;
```

Comment:
"not enough to template" really. It's only one value and it's at the end instead of in the middle, so templating seemed less readable for exactly the same intent